Gifski/Utilities.swift

```diff
@@ -592,6 +592,54 @@ extension AVAsset {
 		imageGenerator.requestedTimeToleranceBefore = .zero
 		return imageGenerator.image(at: time)
 	}
+
+	func trimBlankFrames() -> AVAsset? {
```

Comment:
I think there should be a method on `AVAssetTrack` (with the same name) that contains the main logic and this method should just use it.

Reply:
Makes sense, added in a3a64ed.